tests/test.rs

```diff
@@ -259,17 +259,14 @@ fn test_transform_block_some() -> Result<()> {
     let nodes = parse2_with_config(tokens, config)?;
     let Node::Block(block) = get_element_child(&nodes, 0, 0) else { panic!("expected block") };
 
+    dbg!(block.value.as_ref());
+
     assert_eq!(
         match block.value.as_ref() {
-            Expr::Block(expr) => {
-                match &expr.block.stmts[0] {
-                    Stmt::Expr(Expr::Lit(expr)) => match &expr.lit {
-                        Lit::Str(lit_str) => Some(lit_str.value()),
-                        _ => None,
-                    },
-                    _ => None,
-                }
-            }
+            Expr::Lit(exp) => match &exp.lit {
```

Comment:
It's looks like it parse to a different thing.
Before it was a block of code. In case of example:
```
{
"percent
}
```

And now only its body was saved
`Expr::Lit`